app/assets/stylesheets/components/_index.scss

```diff
@@ -4,3 +4,6 @@
 @import "navbar";
 @import "cards";
 
+.container{
```

Comment:
do you need to do that if you delete the container class from the application.html.erb ?

Reply:
i see you use it somewhere else